lib/numbers.md

```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Numbers like whole numbers and decimals are defined as fixnum or float numbers.
+For cases involving imaginary or other cases in math, Ruby handles them as seperate
+units as part of the fixnums or floats.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+PEMDAS appears to work as normal in Ruby as it would in any other math setting. 
+There are some commands that can be put into an equation to return a value with 
+or without a decimal.
```

Comment:
I am not sure either.

Its possible that when I read the "common operations and comparisons" that I thought of rounding (or flooring) numbers to make a number whole. Other than that I can't think of other commands I would do besides rounding and PEMDAS.